src/test/java/lotto/domain/LottoGameTest.java

```diff
@@ -0,0 +1,20 @@
+package lotto.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class LottoGameTest {
+	LottoGame lotto = new LottoGame();
+
+	@DisplayName("로또 구매 개수 계산 테스트")
+	@ParameterizedTest
+	@CsvSource({"14000,14", "1000,1", "10000,10"})
```

Comment:
```suggestion
	@CsvSource({"14000,14", "1000,1", "10000,10", "0,0", "999,0", "-1000,0"})
```
위 테스트도 통과하게 수정해보면 좋겠네요

Reply:
네 해당 처리 로직은 추가하였습니다.